liquibase-core/src/test/java/liquibase/database/core/PostgresDatabaseTest.java

```diff
@@ -1,20 +1,27 @@
 package liquibase.database.core;
 
+import liquibase.Scope;
 import liquibase.changelog.column.LiquibaseColumn;
 import liquibase.database.AbstractJdbcDatabaseTest;
 import liquibase.database.Database;
 import liquibase.database.ObjectQuotingStrategy;
 import liquibase.structure.core.Table;
+import liquibase.util.StringUtil;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.nio.charset.Charset;
+
 import static org.junit.Assert.*;
 
 /**
  * Tests for {@link PostgresDatabase}
  */
 public class PostgresDatabaseTest extends AbstractJdbcDatabaseTest {
 
+    private static final long PGSQL_PK_BYTES_LIMIT = 63;
```

Comment:
Use the values from `PostgresDatabase` here to avoid the potential for inconsistencies over time.